packages/patternfly-4/content/design-guidelines/usage-and-behavior/lists-and-tables/design.md

```diff
@@ -67,20 +67,20 @@ The expandable list adds an expansion panel to every row to reveal more details
 Use an expandable data list when you have more information than will comfortably fit inside a row, or you want to provide a way for advanced users to access information that is not applicable to all users.
 
 ## Data tables
+In general, PatternFly supports several variations of the data table component. Here are where they differentiate the most.
```

Comment:
This feels a bit awkward. Maybe say, "PatternFly supports several variations of the data table component. These include:"  and then list them. Since you added Sortable Table as a new variant, shouldn't that be added here?

Reply:
Okay yeah I'll change that - I wasn't sure if that was the intent with this introductory sentence because I don't think it is written that way with data lists. I can list types of data lists as well though so they are consistent